controller/src/main/java/org/jboss/as/controller/ModelController.java

```diff
@@ -40,10 +41,10 @@
 public interface ModelController {
 
     /**
-     * A {@link RuntimePermission} needed to access a {@link ModelController} via {@link org.jboss.msc.service.Service#getValue()} or
-     * to invoke its methods. The name of the necessary {@code RuntimePermission} is "{@code canAccessModelController}."
+     * A {@link org.jboss.as.controller.security.ControllerPermission} needed to access a {@link ModelController} via {@link org.jboss.msc.service.Service#getValue()} or
+     * to invoke its methods. The name of the permission is "{@code canAccessModelController}."
      */
-    RuntimePermission ACCESS_PERMISSION = new RuntimePermission("canAccessModelController");
+    ControllerPermission CAN_ACCESS_MODEL_CONTROLLER = new ControllerPermission("canAccessModelController");
```

Comment:
Please revert the name change. It's not necessary and this is public API. The type change is unavoidable, but it's likely anyone using this constant would only care about the "Permission"  type anyway.